py/circuitpy_defns.mk

```diff
@@ -833,7 +833,8 @@ $(BUILD)/lib/libm/kf_rem_pio2.o: CFLAGS += -Wno-maybe-uninitialized
 # Fetch only submodules needed for this particular port.
 .PHONY: fetch-port-submodules
 fetch-port-submodules:
-	$(TOP)/tools/fetch-submodules.sh data extmod frozen lib $(CURDIR)
+	echo $(basename $(CURDIR))
```

Comment:
maybe remove this echo?

Reply:
Oops, yes, that was to prove that GNU Make basename is not the same /usr/bin/basename.